camviewer.pyw

```diff
@@ -76,6 +78,17 @@ if __name__ == "__main__":
     if options.scale is not None:
         os.environ["QT_SCALE_FACTOR"] = options.scale
 
+    if options.min_timeout is None:
+        # 1 day
+        min_timeout = 24 * 60 * 60
```

Comment:
I was only intending these as debug tools. I'm also happy to remove them.

Reply:
I guess the second intention for these is to allow us to tune the timeout bounds in https://github.com/pcdshub/engineering_tools/blob/master/scripts/camViewer without needing to make a new release.

Reply:
I'm not super concerned with end users setting these parameters incorrectly because most people use the wrapper script to launch the app and once you get into the ui it's pretty clear if you've made a mistake with your timeout numbers.